CONTRIBUTING.rst

```diff
@@ -0,0 +1,16 @@
+Thanks for considering submitting a pull request, we really appreciate that.
+Before doing so, here are a few guidelines:
+
+* You agree to license your contributions under the
+  [license](https://github.com/mozilla/kinto/blob/master/LICENSE).
+* Report bugs as [new issues on
+  github](https://github.com/Kinto/kinto/issues/new).
+* Use pull-requests early so it's open for discussion, even if your
+  contribution isn't ready yet.
+* A pull-request should include tests.
```

Comment:
All pull requests should include tests, they help us avoiding regressions in our code.